### PR TITLE
Chore: Bundle Log Viewer Package

### PR DESCRIPTION
**Pull request by madsrasmussen**
See https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/1977 for more details about the bundling changes.

This PR bundles the Log Viewer Package including all modules.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -137,6 +137,7 @@
 		"./src/packages/documents",
 		"./src/packages/health-check",
 		"./src/packages/language",
+		"./src/packages/log-viewer",
```
Comment (bjarnef) [inline package.json:140]: @madsrasmussen ideally `documents`, `members`, `packages`, `property-editors`, `tags` should be in singular form as well?

Comment (madsrasmussen) [reply package.json:140]: Hi Hi,

Both yes and no 😄 Ideally I would like a name for the "Documents" package that indicates that it covers more than just documents. It includes everything needed for "Document Management" - Documents, Document Types, Blueprints, etc. The obvious one would be to call it Document Management but then we have to add "Management" to many of the packages which also feels a bit off.

If you have any suggestions we are all ears 😄

Comment (madsrasmussen) [reply package.json:140]: But everything inside a package should be singular. I can see we still miss to rename folders here and there.

Comment (bjarnef) [reply package.json:140]: Ahh okay 😅
Hmm 🤔 `document-editor`, `document-editing` .. but I guess it is same challenge as with `management`.

However I think some of the other packages also cover many different parts like `core` which isn't named `cores` 😁

Comment (madsrasmussen) [reply package.json:140]: Yes, you are correct. I just wanted to avoid renaming it twice but it might be better than the inconsistent naming 👍

**Review by leekelleher — APPROVED** [review]

**Pull request merged**